tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -53,7 +53,7 @@
 
     private static final boolean DEBUG = false;
 
-    private static final boolean SHOW_SUBTYPES = true; // CldrUtility.getProperty("SHOW_SUBTYPES", "false").equals("true");
+    private static final boolean SHOW_SUBTYPES = false;
```

Comment:
per ticket: "C. Priority Items: Remove the detailed Errors/Warnings (leave under a flag, so we can enable if we want later)"

Reply:
there was already a flag, just make it false